src/main/java/com/scalar/database/transaction/consensuscommit/CommitHandler.java

```diff
@@ -18,21 +18,36 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-/** */
+/** Commit snapshots to {@link DistributedStorage} */
```

Comment:
I think this class handles the commit process of a transaction, not just store updated values to the storage.

Reply:
Thank you! Fixed.